modules/ossm-rn-new-features.adoc

```diff
@@ -337,6 +381,28 @@ Kiali 1.36 includes the following features and enhancements:
 
 This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
 
+=== Component versions included in {SMProductName} version 2.0.11
```

Comment:
This section doesn't appear in your output (there are duplicate 2.0.10 headings). Do you need to regenerate and resync your preview?

Also, are we really shipping a -dev version of Envoy?

Reply:
Thanks, @JStickler! I resynced, generated a new preview build, and confirmed the output.

I also posted a question in slack to try to confirm if the -dev version is correct.

Reply:
@rh-tokeefe thanks, it just seemed weird to me that we'd be delivering a -dev version when there are multiple production versions available.

Reply:
@JStickler to more than just you, I assure you.  =)

I just made the update to list it as 1.14.5 per the slack discussion. I've squashed and generated a new preview build.